[4.6] Doc for Elasticsearch 8 (#3022)
ezsystems/developer-documentation: docs/update_and_migration/from_4.6/update_from_4.6.md

No additional steps needed.

## v4.6.27

### Elasticsearch 8 support

As of v4.6.27, [[= product_name =]] adds optional support for Elasticsearch 8.19 or higher through the new `ibexa/elasticsearch8` package.

By default, [[= product_name =]] continues to support Elasticsearch 7.16.2+ with the `ibexa/elasticsearch` package.

> If you want to use Elasticsearch 8, follow these steps:

Reviewer: ```suggestion
To use Elasticsearch 8, follow these steps:
```

> #### Install the Elasticsearch 8 package

Reviewer: ```suggestion
#### Install Elasticsearch 8 package
```

Replace the existing Elasticsearch package and install Elasticsearch 8:

```bash
composer require ibexa/elasticsearch8:[[= latest_tag_4_6 =]]
```

#### Update Elasticsearch server

Upgrade your Elasticsearch server to version 8.19 or higher.

> Follow the [Elasticsearch upgrade guide](https://www.elastic.co/guide/en/elastic-stack/8.19/upgrading-elastic-stack.html#prepare-to-upgrade) for detailed instructions.

Reviewer: ```suggestion
For detailed instructions, follow the [Elasticsearch upgrade guide](https://www.elastic.co/guide/en/elastic-stack/8.19/upgrading-elastic-stack.html#prepare-to-upgrade).
```

Preferential

> When using [[= product_name_cloud =]], see [Elasticsearch service](https://docs.upsun.com/add-services/elasticsearch.html) for a list of supported versions.

Reviewer: ```suggestion
When you use [[= product_name_cloud =]], see [Elasticsearch service](https://docs.upsun.com/add-services/elasticsearch.html) for a list of supported versions.
```

AFAIR, we tend to avoid the gerund in similar statements.
